File: atcoder/abc054/a.py
import sys
sys.setrecursionlimit(10000000)
input=lambda : sys.stdin.readline().rstrip()

d=[2,3,4,5,6,7,8,9,10,11,12,13,1]
a,b=map(int,input().split())
print('Alice' if d.index(a)>d.index(b) else 'Draw' if d.index(a)==d.index(b) else 'Bob')

File: atcoder/abc054/a.hs
getInts = do
    x <- getLine
    let y = map (read::String->Integer) (words x) 
    return y

main::IO()
main=do
    [a,b] <- getInts
    let (x,y) = ((a+11)`mod`13,(b+11)`mod`13)
    putStrLn$if x==y then "Draw" else if x>y then "Alice" else "Bob"
